packages/config/lib/manager.js

```diff
@@ -117,7 +128,8 @@ class ConfigManager extends EventEmitter {
     }
     const fullEnv = {
       ...this.env,
-      [PLT_ROOT]: join(this.fullPath, '..')
+      [PLT_ROOT]: join(this.fullPath, '..'),
+      ...process.env
```

Comment:
I think you want to put it at the beggining. Otherwise it might overwrite some env vars from config.env.